app/src/main/java/net/gsantner/markor/format/FormatRegistry.java

```diff
@@ -88,41 +91,29 @@ public static class Format {
         }
     }
 
-    public static final Format[] FORMATS = new Format[]{
+    // Order here is used to **determine** format by it's file extension and/or content heading
+    public static final List<Format> FORMATS = Arrays.asList(
             new Format(FormatRegistry.FORMAT_MARKDOWN, R.string.markdown, ".md", CONVERTER_MARKDOWN),
-            new Format(FormatRegistry.FORMAT_PLAIN, R.string.plaintext, ".txt", CONVERTER_PLAINTEXT),
+            new Format(FormatRegistry.FORMAT_CSV, R.string.csv, ".csv", CONVERTER_CSV),
             new Format(FormatRegistry.FORMAT_TODOTXT, R.string.todo_txt, ".todo.txt", CONVERTER_TODOTXT),
             new Format(FormatRegistry.FORMAT_WIKITEXT, R.string.wikitext, ".txt", CONVERTER_WIKITEXT),
+            new Format(FormatRegistry.FORMAT_KEYVALUE, R.string.key_value, ".yaml", CONVERTER_KEYVALUE),
             new Format(FormatRegistry.FORMAT_ASCIIDOC, R.string.asciidoc, ".adoc", CONVERTER_ASCIIDOC),
-            new Format(FormatRegistry.FORMAT_CSV, R.string.csv, ".csv", CONVERTER_CSV),
-            new Format(FormatRegistry.FORMAT_ORGMODE, R.string.orgmode, ".org", CONVERTER_ORGMODE),
+            new Format(FormatRegistry.FORMAT_PLAIN, R.string.plaintext, ".txt", CONVERTER_PLAINTEXT),
             new Format(FormatRegistry.FORMAT_EMBEDBINARY, R.string.embed_binary, ".jpg", CONVERTER_EMBEDBINARY),
-            new Format(FormatRegistry.FORMAT_UNKNOWN, R.string.none, "", null),
-    };
-
-
-    // Order here is used to **determine** format by it's file extension and/or content heading
-    private final static TextConverterBase[] CONVERTERS = new TextConverterBase[]{
-            CONVERTER_MARKDOWN,
-            CONVERTER_CSV,
-            CONVERTER_TODOTXT,
-            CONVERTER_WIKITEXT,
-            CONVERTER_KEYVALUE,
-            CONVERTER_ASCIIDOC,
-            CONVERTER_PLAINTEXT,
-            CONVERTER_EMBEDBINARY,
-            CONVERTER_ORGMODE,
-    };
+            new Format(FormatRegistry.FORMAT_ORGMODE, R.string.orgmode, ".org", CONVERTER_ORGMODE),
```

Comment:
Effective this is also the check order. To my understanding the last 3 have to be in order EMBEDBINARY,PLAIN,NONE, and everything else above. OrgMode would be first priority detected as PLAINTEXT as far I understand.